fe/fe-core/src/main/java/com/starrocks/feature/ProductFeature.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.feature;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductFeature {
+    private String name;
+    private String version;
+    private String description;
+    private String link;
+
+    public ProductFeature(String name, String version, String description, String link) {
+        this.name = name;
+        this.version = version;
+        this.description = description;
+        this.link = link;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getVersion() {
+        return version;
+    }
+
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getLink() {
+        return link;
+    }
+
+    // please write your feature by creating a final static Feature object
+
+    // get all features
+    public static List<ProductFeature> getFeatures() {
+        List<ProductFeature> features = new ArrayList<>();
```

Comment:
features should be static final for each release, no need to new everytime this interface get called.